packages/next/server/web/spec-extension/cookies.ts

```diff
@@ -55,6 +55,9 @@ export class Cookies extends Map<string, string> {
       )
     )
   }
+  [Symbol.for('edge-runtime.inspect.custom')]() {
+    return Object.fromEntries(this.entries())
```

Comment:
I think we can update `@edge-runtime/format` to do this by default for serializing a `Map`